resources/gui/cosmoscout.html

```diff
@@ -0,0 +1,546 @@
+                    <i class="material-icons">expand_more</i>
+                </a>
+            </span>
+
+            <span id="increaseControl" class="item">
+                <a class="btn light-glass" id="increase-day-button" data-diff="86400">
+                    <i class="material-icons">expand_less</i>
+                </a>
+                <a class="btn light-glass" id="increase-month-button" data-diff="2628000">
+                    <i class="material-icons">expand_less</i>
+                </a>
+                <a class="btn light-glass" id="increase-year-button" data-diff="31536000">
+                    <i class="material-icons">expand_less</i>
+                </a>
+                <a class="btn light-glass" id="increase-hour-button" data-diff="3600">
+                    <i class="material-icons">expand_less</i>
+                </a>
+                <a class="btn light-glass" id="increase-minute-button" data-diff="60">
+                    <i class="material-icons">expand_less</i>
+                </a>
+                <a class="btn light-glass" id="increase-second-button" data-diff="1">
+                    <i class="material-icons">expand_less</i>
+                </a>
+            </span>
+
+            <span id="speed-decrease-button-container" class="item">
+                <a class="btn light-glass" data-toggle="tooltip-bottom" title='Fast rewind' id="speed-decrease-button">
+                    <i class="material-icons">fast_rewind</i>
+                </a>
+            </span>
+
+            <span id="speed-increase-button-container" class="item">
+                <a class="btn light-glass" data-toggle="tooltip-bottom" title='Fast forward' id="speed-increase-button">
+                    <i class="material-icons">fast_forward</i>
+                </a>
+            </span>
+        </div>
+
+        <div id="plugin-buttons">
+            <a class="btn light-glass"
+               data-toggle="tooltip-bottom"
+               title="Turn north up"
+               data-call="navigate_north_up">
+                <span id="compass-arrow"></span>
+            </a>
+            <a class="btn light-glass"
+               data-toggle="tooltip-bottom"
+               title="Fix horizon"
+               data-call="navigate_fix_horizon">
+                <i class="material-icons">terrain</i>
+            </a>
+            <a class="btn light-glass"
+               data-toggle="tooltip-bottom"
+               title="Launch to orbit"
+               data-call="navigate_to_orbit">
+                <i class="material-icons">flight_takeoff</i>
+            </a>
+            <a class="btn light-glass"
+               data-toggle="tooltip-bottom"
+               title="Land on surface"
+               data-call="navigate_to_surface">
+                <i class="material-icons">flight_land</i>
+            </a>
+        </div>
+    </div>
+
+    <div id="left-side-bar" class="side-bar left">
```

Comment:
For consistency, you could remove the .side-bar and .left classes here. ` <div id="side-bar">` should be sufficient; we will not have a right sidebar anytime soon.

Reply:
Updated. For consistency `side-bar` got renamed to `sidebar`.